spec/requests/admin/feature_flags_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-describe 'GET admin/feature-flags' do
+describe 'GET admin/feature-flags', openapi: false do
```

Comment:
do you need this? Since you changed the file location

Reply:
BTW it would be great to document `openapi: false` skips the endpoint doc generation

Reply:
yes, because even though we changed it, it's still a safeguard because this specs breaks the docs and we don't really want them in the API docs either